models/lfs.go

```diff
@@ -0,0 +1,98 @@
+package models
+
+import (
+	"errors"
+	"github.com/go-xorm/xorm"
+	"time"
+)
+
+type LFSMetaObject struct {
```

Comment:
Not sure what you mean, technically with LFS there's currently only the v1 legacy API, v1 batch API and the v1.3 batch API (which is only a slightly extended version of v1 batch). The LFSMetaObject model should fit all API versions currently specified.